toolchain/gcc.py

```diff
@@ -52,13 +52,12 @@ def setUp(self):
                              'gmp-devel', 'glibc-devel', 'mpfr-devel',
                              'makeinfo', 'texinfo', 'mpc-devel'])
         else:
-            packages.extend(['glibc-static', 'autogen', 'guile',
-                             'guile-devel', 'libgo', 'libgo-devel',
-                             'libgo-static', 'elfutils-devel',
+            packages.extend(['glibc-static', 'elfutils-devel',
                              'texinfo-tex', 'texinfo', 'elfutils-libelf-devel',
                              'gmp-devel', 'mpfr-devel', 'libmpc-devel',
-                             'gcc-gnat', 'libgnat', 'zlib-devel',
-                             'gettext', 'libgcc', 'libgomp'])
+                             'zlib-devel', 'gettext', 'libgcc', 'libgomp'])
+            if dist.name == 'rhel8':
```

Comment:
Commit message says `the changes applicable are for both RHEL8.6 and RHEL9`, but the above change is for the whole RHEL8. If you want this change has to apply only to 8.6+ then please change the above statement accordingly. And where is RHEL9 related change?
Thanks!!

Reply:
@preetithakur is this worked for RHEL8.5 and below versions?
if yes and only issue with rhel8.6 and 9.0 then we have to skip for those lower Distro version.

Reply:
@Naresh-ibm yes Naresh :)  @preetithakur need to send patch for same